cpp/bench/ann/CMakeLists.txt

```diff
@@ -124,6 +125,7 @@ function(ConfigureAnnBench)
   target_link_libraries(
     ${BENCH_NAME}
     PRIVATE ${ConfigureAnnBench_LINKS}
+            $<$<BOOL:${CUVS_FAISS_ENABLE_GPU}>:raft::raft>
```

Comment:
Please move this dependency down to the algo-specific configs.

Reply:
Done!